src/clarity-angular/wizard/wizard.ts

```diff
@@ -252,12 +271,20 @@ export class Wizard implements OnInit, OnDestroy, AfterContentInit, DoCheck {
     }
 
     // LEGACY: Naming convention matches old wizard
-    public next(): void {
-        this.navService.next();
+    public next(skipChecksAndEmits: boolean = true): void {
+        if (skipChecksAndEmits) {
```

Comment:
No. Not for how `Wizard.next()` is used 80% of the time. The most common use case is programmatically navigating the wizard when you've prevented default navigation. In that case the event emitters and checks have already run. You don't want to run them again or you'll get an infinite loop of event emissions.